static/app/utils/useCopyToClipboard.tsx

```diff
@@ -0,0 +1,62 @@
+import {useCallback, useRef, useState} from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {t} from 'sentry/locale';
+
+type Opts = {
+  text: string;
+  errorMessage?: string;
+  hideMessages?: boolean;
+  onCopy?: undefined | ((copiedText: string) => void);
+  onError?: undefined | ((error: Error) => void);
+  successMessage?: string;
```

Comment:
Any chance we could add some comments here?

Reply:
done